cmd/cli/job/get.go

```diff
@@ -0,0 +1,103 @@
+package job
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/kubectl/pkg/util/i18n"
+
+	"github.com/bacalhau-project/bacalhau/cmd/util"
+	"github.com/bacalhau-project/bacalhau/cmd/util/flags/cliflags"
+	"github.com/bacalhau-project/bacalhau/cmd/util/flags/configflags"
+	"github.com/bacalhau-project/bacalhau/cmd/util/hook"
+	"github.com/bacalhau-project/bacalhau/pkg/util/templates"
+)
+
+var (
+	getLong = templates.LongDesc(i18n.T(`
+		Get the results of the job, including stdout and stderr.
+`))
+
+	//nolint:lll // Documentation
+	getExample = templates.Examples(i18n.T(`
+		# Get the results of a job.
+		bacalhau job get 51225160-807e-48b8-88c9-28311c7899e1
+
+		# Get the results of a job, with a short ID.
+		bacalhau job get ebd9bf2f
+`))
+)
+
+type GetOptions struct {
+	DownloadSettings *cliflags.DownloaderSettings
+}
+
+func NewGetOptions() *GetOptions {
+	return &GetOptions{
+		DownloadSettings: cliflags.NewDefaultDownloaderSettings(),
+	}
+}
+
+func NewGetCmd() *cobra.Command {
+	OG := NewGetOptions()
```

Comment:
What would be great is if we can have the existing `get` command call this new implementation to avoid duplicating code. Similar to how we are handling `bacalhau logs` and `bacalhau job logs` https://github.com/bacalhau-project/bacalhau/blob/main/cmd/cli/logs/logs.go

I would also move the tests to this package as our goal is to eventually deprecate `bacalhau get` and only keep `bacalhau job get`

Reply:
Can you check now, I have removed the duplicate code.